apps/full-site-editing/full-site-editing-plugin/wpcom-block-editor-nav-sidebar/src/components/nav-item/index.tsx

```diff
@@ -20,36 +21,42 @@ interface NavItemProps {
 	statusLabel?: string;
 }
 
-export default function NavItem( { item, postType, selected, statusLabel }: NavItemProps ) {
-	const buttonClasses = classNames( 'wpcom-block-editor-nav-item', {
-		'is-selected': selected,
-	} );
+const NavItem = forwardRef< HTMLLIElement, NavItemProps >(
+	( { item, postType, selected, statusLabel }, ref ) => {
+		const buttonClasses = classNames( 'wpcom-block-editor-nav-item', {
+			'is-selected': selected,
+		} );
 
-	const titleClasses = classNames( 'wpcom-block-editor-nav-item__title', {
-		'is-untitled': ! item.title?.raw,
-	} );
+		const titleClasses = classNames( 'wpcom-block-editor-nav-item__title', {
+			'is-untitled': ! item.title?.raw,
+		} );
 
-	const defaultEditUrl = addQueryArgs( 'post.php', { post: item.id, action: 'edit' } );
-	const editUrl = applyFilters(
-		'a8c.WpcomBlockEditorNavSidebar.editPostUrl',
-		defaultEditUrl,
-		item.id,
-		postType.slug
-	);
+		const defaultEditUrl = addQueryArgs( 'post.php', { post: item.id, action: 'edit' } );
+		const editUrl = applyFilters(
+			'a8c.WpcomBlockEditorNavSidebar.editPostUrl',
+			defaultEditUrl,
+			item.id,
+			postType.slug
+		);
 
-	return (
-		<li>
-			<Button
-				className={ buttonClasses }
-				href={ editUrl }
-				target={ applyFilters( 'a8c.WpcomBlockEditorNavSidebar.linkTarget', undefined ) }
-			>
-				<div className={ titleClasses }>
-					{ item.title?.raw ||
-						_x( 'Untitled', 'post title for posts with no title', 'full-site-editing' ) }
-				</div>
-				{ statusLabel && <div className="wpcom-block-editor-nav-item__label">{ statusLabel }</div> }
-			</Button>
-		</li>
-	);
-}
+		return (
+			<li ref={ ref }>
+				<Button
+					className={ buttonClasses }
+					href={ editUrl }
+					target={ applyFilters( 'a8c.WpcomBlockEditorNavSidebar.linkTarget', undefined ) }
+				>
+					<div className={ titleClasses } title={ item.title?.raw }>
+						{ item.title?.raw ||
+							_x( 'Untitled', 'post title for posts with no title', 'full-site-editing' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **40** times:
`translate( 'Untitled', { context: 'post title for posts with no title'} )` *ES Score: 15*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/nav-sidebar-style)